### PR TITLE
Revamp AWS Quickstarts with DevServices

### PR DESCRIPTION
**Pull request by scrocquesel**
**Check list**:

Your pull request:

- [x] targets the `development` branch
- [x] uses the `999-SNAPSHOT` version of Quarkus
- [x] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [x] has integration/native tests (`mvn clean verify -Pnative`)
- [x] makes sure the associated guide must not be updated
- [x] links the guide update pull request (if needed)
- [x] updates or creates the `README.md` file (with build and run instructions)
- [x] for new quickstart, is located in the directory _component-quickstart_
- [x] for new quickstart, is added to the root `pom.xml` and `README.md`

### CODE REVIEW

File: amazon-dynamodb-quickstart/src/main/docker/Dockerfile.jvm [hunk 1]
```diff
@@ -7,48 +7,91 @@
 #
 # Then, build the image with:
 #
-# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/amazon-dynamodb-jvm .
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/amazon-ssm-quickstart-jvm .
 #
 # Then run the container using:
 #
-# docker run -i --rm -p 8080:8080 quarkus/amazon-dynamodb-jvm
+# docker run -i --rm -p 8080:8080 quarkus/amazon-ssm-quickstart-jvm
```
Comment (gsmet) [inline amazon-dynamodb-quickstart/src/main/docker/Dockerfile.jvm:14]: These changes are off.

How did you update these Dockerfiles? We should be updating them automatically on release.

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks for that but changes to the Dockerfiles look incorrect.

**Comment by scrocquesel** [comment]
@gsmet kind reminder

**Review by gsmet — APPROVED** [review]
Sorry it slipped, merging now.

Thanks for this awesome work!

**Pull request merged**